Makefile

```diff
@@ -0,0 +1,55 @@
+.PHONY: build lint test clean run_image build_image download-vectors clean-vectors \
+	setup-hive test-pattern-default run-hive run-hive-debug clean-hive-logs
+
+build:
+	cargo build --workspace
+
+lint:
+	cargo clippy --all-targets --all-features --workspace -- -D warnings
+
+CRATE ?= *
+test:
+	cargo test -p '$(CRATE)'
+
+clean:  clean-vectors
+	cargo clean
+	rm -rf hive
+
+run_image: build_image
+	docker run --rm -p 127.0.0.1:8545:8545 ethereum_rust --http.addr 0.0.0.0
+
+build_image:
+	docker build -t ethereum_rust .
+
+SPECTEST_VERSION := v3.0.0
+SPECTEST_ARTIFACT := tests_$(SPECTEST_VERSION).tar.gz
+SPECTEST_VECTORS_DIR := cmd/ef_tests/vectors
+
+download-vectors: clean-vectors
```

Comment:
instead of having clean-vectors as a dependency, we could leverage the build system like we do here https://github.com/lambdaclass/lambda_ethereum_consensus/blob/main/Makefile#L224 so that we only download them if they are not present or if spectest version changed?

Reply:
Updated.